site/docs/v1/tech/apis/hosted-backend.adoc

```diff
@@ -54,6 +54,15 @@ FusionAuth will set the domain on these cookies to `.example.com` where `example
 
 What this means is that FusionAuth needs to be hosted on the same domain or a subdomain or sibling domain of the application that you intend to use with these endpoints.
 
+[WARNING]
+====
+When developing against a FusionAuth cloud instance with a `.fusionauth.io` address, unless your application also lives at a `.fusionauth.io` address, you'll receive a `403` response code. This is because the hosted backend uses `SameSite` cookies, so if a request comes in that isn't begun from the same domain, it will fail.
```

Comment:
The `403` is not due to our cookies, but instead a validation step we added. 

https://github.com/FusionAuth/fusionauth-app/blob/a091a509d5486ecaf4c45c37a76ea27f2d6dced0/src/main/java/io/fusionauth/app/action/app/BaseAppAction.java#L105

The intention of failing early is to let a developer know that this won't work unless they develop in the same domain.

I don't know that it is necessary to mention the `SameSite` cookie attribute here. We don't actually use `SameSite` cookies, instead we set an attribute on a cookie called `SameSite` which instructs the browser when to send the cookie to FusionAuth as it relates to cross origin request. 

The bigger message is that if you are developing across origin, you will have problems.